backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -273,6 +276,32 @@ func (c *workflowCompiler) addContainerExecutorTemplate() string {
 			Env:     commonEnvs,
 		},
 	}
+	caBundleCfgMapName := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_NAME")
+	caBundleCfgMapKey := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_KEY")
+	caBundleMountPath := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_MOUNTPATH")
```

Comment:
Let's rename these, "artifact copy step" is more relevant to v1, let's use: 

* `EXECUTOR_CABUNDLE_CONFIGMAP_NAME`
* `EXECUTOR_CABUNDLE_CONFIGMAP_KEY`
* `EXECUTOR__CABUNDLE_MOUNTPATH`